packages/cli/__tests__/zosfiles/__unit__/copy/ds/Ds.handler.unit.test.ts

```diff
@@ -234,7 +237,79 @@ describe("DsHandler", () => {
         const result = await promptFn(commandParameters.arguments.toDataSetName);
 
         expect(promptMock).toHaveBeenCalledWith(
-            `The dataset '${toDataSetName}' exists on the target system. This copy will result in data loss.` +
+            `The dataset '${toDataSetName}' exists on the target system. This copy may result in data loss.` +
+            ` Are you sure you want to continue? [y/N]: `
+        );
+        expect(result).toBe(false);
+    });
+    it("should prompt the user about duplicate member names and return true when input is 'y", async () => {
+        const handler = new DsHandler();
+
+        expect(handler).toBeInstanceOf(ZosFilesBaseHandler);
+        const fromDataSetName = "ABCD";
+        const toDataSetName = "EFGH";
+        const enq = "SHR";
+        const replace = false;
+        const safeReplace = false;
+        const responseTimeout: any = undefined;
+
+        const commandParameters: any = {
+            arguments: {
+                fromDataSetName,
+                toDataSetName,
+                enq,
+                replace,
+                safeReplace,
+                responseTimeout
+            },
+            response: {
+                console: { promptFn: jest.fn() }
+            }
+        };
+        const promptMock = jest.fn();
+        promptMock.mockResolvedValue("y");
+
+        const promptForDuplicates = (handler as any)["promptForLikeNamedMembers"]({ prompt: promptMock });
+        const result = await promptForDuplicates();
+
+        expect(promptMock).toHaveBeenCalledWith(
+            `The source and target data sets have identical member names. The contents of those members will be overwritten.` +
```

Comment:
I think it would be clearer to say which members will be overwritten: "The contents of the target members will be ..."